pynest/examples/EI_clustered_network/helper.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+#
+# helper.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""PyNEST EI-clustered network: Helper Functions
+------------------------------------------------
+
+Helper functions to calculate synaptic weights to construct
+random balanced networks and plot raster plot with color groups.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+small = 1e-10
```

Comment:
Add a comment explaining what this parameter is used for. Constants should be uppercase according to PEP8, so I suggest to rename to `SMALL`.

Reply:
I forgot to remove it. It was used in a function which is not necessary anymore.